### ISSUE TITLE
Fix/enhance validation messages on repeating groups

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#1276]
### Description

With the implementation of `minCount` on repeating groups, we need to be able to inform the user in an clear an accessible way. This might be an opportunity to look into how validation behaves and looks on repeating groups. 

## Case 1 - Correct feedback to the user
The validation text need to reference the field it is validating. How do we solve this with groups where there may not be an identifying text to use.
![image (2)](https://github.com/Altinn/app-frontend-react/assets/33222679/5a893241-95f5-4210-b54b-b4ddc919c80d)

## Case 2 - Validate the whole group
If there are a `minCount= 2`and the user has created one group. We only have a way to validate on groups. Maybe we need a way to show visually that it is missing a group.

## Case 3 - Difference between open and closed groups
We might need to ways of showing validations based on if the group is open when the validation happens.

**Comment by rvessb** [comment]
Problemet med å vise feil på gruppa ( i dette tilfellet for få rader) ligner jo litt på utfordringen beskrevet i #912 , men i tillegg trenger man her å koble inn verdier i feilmeldingen fra den repeterende datastrukturen. Hva med å benytte teksten som man kan legge på "legg til ny rad-knappen" som en del av feilmeldingen?
![image](https://github.com/Altinn/app-frontend-react/assets/43333852/28be1296-cac7-4a9a-9ae1-0ccf47ddd686)

**Comment by Febakke** [comment]
Ja, enig i dette forslaget @rvessb 👌

Om vi har mulighet kunne vi jobbet litt med den feilmeldingsteksten slik at den virker mindre generert. Kanskje noe som: "Det må legges til minst en kommune for Ringdue" Jeg skal i forbindelse med denne oppgaven prøve å få litt hjelp av noen som er sterkere på gode teksten enn det jeg er.

**Comment by olemartinorg** [comment]
Det eksempelet fungerer fint for entall, men hvis `minCount` er satt til 2 blir jo teksten `Minst 2 kommune for Ringdue er påkrevd`. Da må eventuelt også innføre forskjellige tekstnøkler for entall/flertall her.

Det holder heller ikke helt mål for alle språk: https://localizely.com/language-plural-rules/

**Comment by xmrsa** [comment]
Hvis vi kunne fått en tilsvarende løsning for dette, som med standard valideringer for obligatoriske felt, så hadde det jo fungert bedre. Hvis man har automatisk tekst basert på "Legg til + ..." som @rvessb foreslår, men så i tillegg har en parameter (tilsvarende "requiredValidation"), man kan sette med tekst fra resource-fil, dersom man krever mer enn ett element i den repeterende strukturen.

**Comment by olemartinorg** [comment]
Enig i at noe tilsvarende `requiredValidation` heller burde lages, ja.

> Det beste hadde vært om man kunne få satt inn en tom rad automatisk

Vi har støtte for det med `openByDefault`, men her mener du kanskje at vi automatisk legger til en tom rad så snart brukeren har begynt å fylle ut den siste raden - slik at det alltid er en ny "fersk" rad tilgjengelig under? Det kan vi jo lage, men det betyr jo også at vi må rydde bort slike tomme rader fra slutten på et tidspunkt, eller bare vise en tom rad til brukeren (uten å faktisk legge den inn i datamodellen før brukeren fyller noe ut i den), men det er jo litt av det motsatte vi prøver å gjøre for å levere #555.
